{
  "id": "oval:org.opensuse.security:def:20141912",
  "version": "1",
  "class": "vulnerability",
  "metadata": {
    "Title": "CVE-2014-1912",
    "affected": {
      "family": "unix",
      "Platform": [
        "openSUSE Leap 42.1"
      ]
    },
    "reference": [
      {
        "ref_id": "CVE-2014-1912",
        "ref_url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-1912",
        "source": "CVE"
      }
    ],
    "Description": "\n    Buffer overflow in the socket.recvfrom_into function in Modules/socketmodule.c in Python 2.5 before 2.7.7, 3.x before 3.3.4, and 3.4.x before 3.4rc1 allows remote attackers to execute arbitrary code via a crafted string.\n    ",
    "advisory": {
      "from": "security@suse.de",
      "Severity": "Important",
      "cve": {
        "text": "CVE-2014-1912",
        "href": "https://www.suse.com/security/cve/CVE-2014-1912.html"
      },
      "bugzilla": [
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 882915",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 863741",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 1049392",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 1049422",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 1049392",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        },
        {
          "text": "SUSE bug 1049422",
          "href": "https://bugzilla.suse.com/CVE-2014-1912"
        }
      ],
      "issued": {},
      "updated": {},
      "affected_cpe_list": {
        "Cpe": null
      }
    }
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "OR",
        "criterias": [
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.opensuse.security:tst:2009138784",
                "comment": "libpython2_7-1_0-2.7.9-21.3 is installed"
              },
              {
                "test_ref": "oval:org.opensuse.security:tst:2009151571",
                "comment": "libpython2_7-1_0 is signed with openSUSE key"
              }
            ]
          },
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.opensuse.security:tst:2009138785",
                "comment": "libpython2_7-1_0-32bit-2.7.9-21.3 is installed"
              },
              {
                "test_ref": "oval:org.opensuse.security:tst:2009151572",
                "comment": "libpython2_7-1_0-32bit is signed with openSUSE key"
              }
            ]
          },
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.opensuse.security:tst:2009138786",
                "comment": "python-base-2.7.9-21.3 is installed"
              },
              {
                "test_ref": "oval:org.opensuse.security:tst:2009151573",
                "comment": "python-base is signed with openSUSE key"
              }
            ]
          },
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.opensuse.security:tst:2009138787",
                "comment": "python-devel-2.7.9-21.3 is installed"
              },
              {
                "test_ref": "oval:org.opensuse.security:tst:2009151574",
                "comment": "python-devel is signed with openSUSE key"
              }
            ]
          },
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.opensuse.security:tst:2009138788",
                "comment": "python-xml-2.7.9-21.3 is installed"
              },
              {
                "test_ref": "oval:org.opensuse.security:tst:2009151575",
                "comment": "python-xml is signed with openSUSE key"
              }
            ]
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.opensuse.security:tst:2009117743",
        "comment": "openSUSE Leap 42.1 is installed"
      }
    ]
  }
}
